website/docs/customization/plugin/plugin_only.md

```diff
@@ -0,0 +1,70 @@
+---
+id: plugin_only
+description: The Plugin Only Mode
+slug: /plugin/plugin_only
+---
+
+# The Plugin-Only Mode
+
+The plugin-only mode is a restricted mode of TaskWeaver that only allows you to use plugins. 
+Compared to the full mode, the plugin-only mode has the following restrictions:
+
+1. The generated code only contains the calls to the plugins. 
+   For example, the following code only calls the `ascii_render` plugin and does not contain any "free-form" code.
+    ```python
+    r1=ascii_render(text="Hello")
+    r1
```

Comment:
In plugin only mode, we allow the reference to the variable `r1`?

Reply:
In theory, yes, but we didn't implement it. So, the answer is no.

Reply:
then, this might be misleading?